target/linux/ath79/nand/base-files/lib/upgrade/platform.sh

```diff
@@ -1,20 +1,27 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0
 #
-# Copyright (C) 2011 OpenWrt.org
+# Copyright (C) 2019 Jeff Kletsky
 #
 
 PART_NAME=firmware
-REQUIRE_IMAGE_METADATA=1
 
+REQUIRE_IMAGE_METADATA=1
 platform_check_image() {
 	return 0
 }
 
+RAMFS_COPY_BIN='fw_printenv fw_setenv'
+RAMFS_COPY_DATA='/etc/fw_env.config /var/lock/fw_printenv.lock'
+
 platform_do_upgrade() {
-	local board=$(board_name)
+	local board="$(board_name)"
```

Comment:
There's no need for this quote because we don't allow compatible string with special characters in the first place.

Reply:
Will be resolved in next push